driver/ni/digital_read.cpp

```diff
@@ -22,129 +22,153 @@
+                                                   "", DAQmx_Val_ChanPerLine));
             LOG(INFO) << "Channel name: " << channel.name;
-        } 
+        }
         LOG(INFO) << "Index channel added to task: " << channel.name;
-        this->numChannels++; 
-        if (err < 0){
-            LOG(ERROR) << "[NI Reader] failed while configuring channel " << channel.name;
+        this->numChannels++;
+        if (err < 0) {
+            LOG(ERROR) << "[NI Reader] failed while configuring channel " << channel.
+                    name;
             this->ok_state = false;
             return -1;
         }
     }
     return 0;
 }
 
-int ni::DigitalReadSource::configureTiming(){
-    if(this->reader_config.timing_source == "none"){ // if timing is not enabled, implement timing in software
-        this->reader_config.period = (uint64_t)((1.0 / this->reader_config.sample_rate) * 1000000000); // convert to microseconds
+int ni::DigitalReadSource::configureTiming() {
+    if (this->reader_config.timing_source == "none") {
+        // if timing is not enabled, implement timing in software
+        this->reader_config.period = (uint64_t) (
+            (1.0 / this->reader_config.sample_rate) *
+            1000000000); // convert to microseconds
         this->numSamplesPerChannel = 1;
-    } else{
+    } else {
         if (this->checkNIError(ni::NiDAQmxInterface::CfgSampClkTiming(this->task_handle,
-                                                                    this->reader_config.timing_source.c_str(),
-                                                                    this->reader_config.sample_rate,
-                                                                    DAQmx_Val_Rising,
-                                                                    DAQmx_Val_ContSamps,
-                                                                    this->reader_config.sample_rate))){
-            LOG(ERROR) << "[NI Reader] failed while configuring timing for task " << this->reader_config.task_name;
+            this->reader_config.timing_source.c_str(),
+            this->reader_config.sample_rate,
+            DAQmx_Val_Rising,
+            DAQmx_Val_ContSamps,
+            this->reader_config.sample_rate))) {
+            LOG(ERROR) << "[NI Reader] failed while configuring timing for task " <<
+                    this->reader_config.task_name;
             this->ok_state = false;
             return -1;
         }
-        this->numSamplesPerChannel = std::floor(this->reader_config.sample_rate / this->reader_config.stream_rate);
+        this->numSamplesPerChannel = std::floor(
+            this->reader_config.sample_rate / this->reader_config.stream_rate);
     }
     this->bufferSize = this->numChannels * this->numSamplesPerChannel;
     return 0;
 }
 
 
-void ni::DigitalReadSource::acquireData(){
-    while(this->running){
+void ni::DigitalReadSource::acquireData() {
+    while (this->running) {
```

Comment:
Breaker?

Reply:
I'll handle this in the next pr dealing with stability issues.